dev_container

```diff
@@ -736,6 +736,8 @@ build_and_run_desc() { c_echo 'Build and run a requested application in a Docker
         --language : Specify the app language implementation to run.
                      Some applications provide both `cpp` and `python` implementations.
         --no_build : Launch the app without attempting to build it first
+        --build_args : Build the app with additional args
+        --build_with : List of optional operators that should be built separated by semicolons (;)
```

Comment:
These additions look good and will make the `build_and_run` command more flexible. Let's clarify that `build_args` values are passed to CMake configuration rather than another build command (such as `./dev_container build`, `./run build`, or `cmake --build`):
```suggestion
        --build_args : Build the app with additional CMake configuration arguments
        --build_with : List of optional operators that should be built separated by semicolons (;)
```